### PR TITLE
Modify the event time

### PR DESCRIPTION
**Pull request by Yu1986**
- Change GMT
- Remove 2020 event

### CODE REVIEW

**Comment by Yu1986** [comment]
@duynguyen please PR review and merge

File: events.md [hunk 1]
```diff
@@ -34,28 +34,25 @@ This session will be divided into two halves, starting with a 15-20 minute prese
...
 - **Join Event:** https://primetime.bluejeans.com/a2m/live-event/dersrzsx
 
 February 25, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
 - **Join Event:** https://primetime.bluejeans.com/a2m/live-event/dersrzsx
 
 March 25, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
```
Comment (duynguyen) [inline events.md:47]: Daylight saving might be an issue in March. Better to check the exact time.

File: events.md [hunk 2]
```diff
@@ -6,23 +6,23 @@
...
 - **Join Event:** https://primetime.bluejeans.com/a2m/live-event/dersrzsx
 
 February 11, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
 - **Join Event:** https://primetime.bluejeans.com/a2m/live-event/dersrzsx
 
 March 11, 2021
-- **Time:** 09:00 - 10:00 PT / 16:00 - 17:00 GMT
+- **Time:** 09:00 - 10:00 PT / 17:00 - 18:00 GMT
```
Comment (duynguyen) [inline events.md:17]: Daylight saving might be an issue in March. Better to check the exact time.

**Review by duynguyen — COMMENTED** [review]
Thanks @Yu1986 . I commented on the March events, for which daylight saving could make the time different. Could you check the exact time?

File: events.md (hunk 2 above)
Comment (Yu1986) [reply events.md:17]: good point, so US summer change in March 14 and Europe change at March 28, so March 25th would be the only one that special

Comment (Yu1986) [reply events.md:17]: good point, verified by https://www.timeanddate.com/worldclock/meetingtime.html?month=5&day=6&year=2021&p1=283&p2=268&iv=0, after March 25th would be 09:00 - 10:00 PT / 16:00 - 17:00 GMT

**Pull request closed without merging**